### PR TITLE
feat: [2U-378] add updates action and error alerts

### PR DESCRIPTION
**Pull request by Thposbsmt**
https://youtrack.raccoongang.com/issue/2U-378
https://youtrack.raccoongang.com/issue/2U-400

### CODE REVIEW

File: src/course-updates/CourseUpdates.jsx [hunk 1]
```diff
@@ -37,88 +44,115 @@ const CourseUpdates = ({ courseId }) => {
...
+              </article>
+            </Layout.Element>
+          </Layout>
+        </section>
+      </Container>
+      <div className="alert-toast">
+        <InternetConnectionAlert
+          isFailed={anyStatusFailed}
+          isQueryPending={anyStatusInProgress || anyStatusPending}
+          onInternetConnectionFailed={() => null}
```
Comment (ruzniaievdm) [inline src/course-updates/CourseUpdates.jsx:152]: Do we need to change the state if there was an error?

Comment (Thposbsmt) [reply src/course-updates/CourseUpdates.jsx:152]: I think no, you mean some specific state or in general?

Comment (ruzniaievdm) [reply src/course-updates/CourseUpdates.jsx:152]: `savingStatus` or `loadingStatus` to `RequestStatus.FAILED` to show error or something else?

**Review by ruzniaievdm — APPROVED** [review]

File: src/constants.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ export const BADGE_STATES = {
 };
 
 export const NOTIFICATION_MESSAGES = {
-  saving: 'saving',
-  duplicating: 'duplicating',
-  deleting: 'deleting',
+  saving: 'Saving',
```
[the remark this answers is not in the document]
Comment (wowkalucky) [reply src/constants.js:22]: There is `Adding` action (outline) as well.

**Review by wowkalucky — CHANGES_REQUESTED** [review]

File: src/constants.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (wowkalucky) [reply src/constants.js:22]: We decided to treat this as a tech debt for now. We'll cover it on the Outline page development.

**Pull request merged**